src/EditorFeatures/Core/Implementation/AutomaticCompletion/BraceCompletionSessionProvider.BraceCompletionSession.cs

```diff
@@ -268,7 +250,13 @@ public void PostReturn()
 
                     if (closingSnapshotPoint.Position > 0 && HasNoForwardTyping(this.GetCaretPosition().Value, closingSnapshotPoint.Subtract(1)))
                     {
-                        _session.AfterReturn(this, CancellationToken.None);
+                        var changesAfterReturn = _session.GetChangesAfterReturn(this, CancellationToken.None);
```

Comment:
`CancellationToken.None`

:(

is thee no cancellation token, or OperationContext that is passed in tha twe can use here?

Reply:
there is nothing passed to us here from the editor :(